document-store/src/integrationTest/java/org/hypertrace/core/documentstore/postgres/PostgresDocStoreTest.java

```diff
@@ -38,6 +46,8 @@ public class PostgresDocStoreTest {
   private static GenericContainer<?> postgres;
   private static Datastore datastore;
   private static String connectionUrl;
+  private static Map<String, Datastore> datastoreMap = Maps.newHashMap();;
```

Comment:
You don't need a datastore map in this class as the test is specific to Postgres only.

Reply:
ok